CONTRIBUTING.md

```diff
@@ -100,6 +100,14 @@ pytest
 pytest tests/${TARGET_TEST_FILE_NAME}
 ```
 
+## Documentation
+Optuna's tutorial is built with [Sphinx-Gallery](https://sphinx-gallery.github.io/stable/index.html) and
+some other requirements like [LightGBM](https://github.com/microsoft/LightGBM) meaning that
+all .py files in `tutorial` directory are run during the documentation build if there's no build cache.
+Whether you edit any tutorial or not doesn't matter.
+
+To avoid the run of tutorial, we recommend that you download build artifacts from our CI and put them to `docs/build`.
```

Comment:
"Recommend" is maybe a bit strong, at least without further guidance to where and what exactly to download.
```suggestion
To avoid having to run the tutorials, you may download build artifacts from our CI and put them in `docs/build`.
```